spec/identity_doc_auth/acuant/request_spec.rb

```diff
@@ -116,9 +116,9 @@
 
     context 'when the request resolves with a 404 status it retries' do
       it 'calls exception_notifier each retry' do
-        allow(subject).to receive(:handle_http_response) do |http_response|
-          http_response
-        end
+        # allow(subject).to receive(:handle_http_response) do |http_response|
+        #   http_response
+        # end
```

Comment:
let's remove this entirely if it's just going to be commented out?
```suggestion
```

Reply:
Yeah, I just missed that one. Thanks.